esphome/components/ina219/ina219.cpp

```diff
@@ -129,6 +129,14 @@ void INA219Component::setup() {
   }
 }
 
+void INA219Component::on_powerdown() {
+  // Mode = 0 -> power down
+  if (!this->write_byte_16(INA219_REGISTER_CONFIG, 0)) {
+    ESP_LOGE(TAG, "INA219 powerdown error");
```

Comment:
```suggestion
    ESP_LOGE(TAG, "powerdown error");
```

TAG already includes the name

Reply:
https://developers.esphome.io/architecture/logging/?h=logging#minimizing-impact